packages/preset-system/package.json

```diff
@@ -12,5 +12,8 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
```

Comment:
I’m not sure about adding dependencies here. Shouldn’t it be a peer dependency instead? We expect people to have core packages installed already.

Reply:
Yep. Let's do that. I guess there's no way we can make it only download for TS users?

Reply:
Yeah a peerDependency is probably more correct; we want the same version that the parent project is using (and if that causes a type error then presenting the error to the user is the right thing to do).